### PR TITLE
Venetian bruhg-fixes

### PR DESCRIPTION
**Pull request by trinkledink**
Venetian Patricians Journal Entry
-Consistent Iannarella Character DNA
-Positivist event after disaster fires if you reformed with positivist help, giving you a positivist intelligentsia
-Polished events, icons, etc.
-Disaster fires from powerful/insurrectionary abolitionist interest group as well

### CODE REVIEW

File: common/character_templates/dvg_agitators.txt [hunk 1]
```diff
@@ -38,3 +38,22 @@
 		chance = 100
 	}
 }
+
+dvg_giancarlo_iannarella_template = {
+	first_name = Giancarlo
+	last_name = Iannarella
+	is_agitator = yes
+	add_character_role = politician
```
Comment (MadJamez) [inline common/character_templates/dvg_agitators.txt:46]: Error: "Wrong type parsed in create_character effect: add_character_role, near line: 46" in file: "common/character_templates/dvg_agitators.txt" near line: 46

**Review by MadJamez — CHANGES_REQUESTED** [review]
Also seeing this errors:
[16:53:36][pdx_persistent_reader.cpp:252]: Error: "Wrong type parsed in create_character effect: add_character_role, near line: 1294" in file: "events/dvg_venice.txt" near line: 1302
[16:53:36][pdx_persistent_reader.cpp:252]: Error: "Wrong type parsed in create_character effect: add_character_role, near line: 1338" in file: "events/dvg_venice.txt" near line: 1344
[16:53:36][pdx_persistent_reader.cpp:252]: Error: "Wrong type parsed in create_character effect: add_character_role, near line: 1598" in file: "events/dvg_venice.txt" near line: 1604

File: common/character_templates/dvg_agitators.txt (hunk 1 above)
Comment (trinkledink) [reply common/character_templates/dvg_agitators.txt]: Okay, I fixed them all by moving them into on_created = { } within the create_character = { }

File: common/character_templates/dvg_agitators.txt [hunk 2]
```diff
@@ -38,3 +38,21 @@
...
+dvg_giancarlo_iannarella_template = {
+	first_name = Giancarlo
+	last_name = Iannarella
+	is_agitator = yes
+	female = no
+	dna = dna_Iannarella
+	age = 31
+	culture = cu:venetian
+	interest_group = ig_intelligentsia
+	ideology = ideology_positivist
```
Comment (MadJamez) [inline common/character_templates/dvg_agitators.txt:51]: Failed to find a valid event target link 'ideology_positivist' at  file: events/dvg_venice.txt line: 1959

**Review by MadJamez — CHANGES_REQUESTED** [review]

**Review by MadJamez — APPROVED** [review]

**Pull request merged**